Defs/Tutor/Concepts_NotedOpportunistic.xml

```diff
@@ -173,5 +173,12 @@
     <priority>50</priority>
     <helpText>After getting 2.2 weapon handling, which is 8 shooting skill with 100% manipulation, you can target body part areas of targets in the open.</helpText>
   </ConceptDef>
+
+  <ConceptDef>
+    <defName>CE_TargettedModesMelee</defName>
+    <label>CE: Targetting in melee</label>
+    <priority>50</priority>
+    <helpText>After getting 5 melee skill it is possible to target body part areas in melee fighting. After reaching 16 melee skill it also becomes possible to aim for exact specific body parts in those areas.\n\ when targetting specific body part (part, not height), the selected part won't be always hit. The chance to actually hit it is 20% * (Pawn melee skill - 15), if the attack "misses" it hits selected height area.</helpText>

```

Comment:
So the chance to hit the target area hits 100% at skill 20.  That would seem to be a bit on the strong side.

Also, the skill of the victim doesn't matter, which is likely not ideal too.